Robocopy/Task/task.json

```diff
@@ -44,7 +44,7 @@
 		},
 		{
 			"name": "files",
-			"type": "string",
+			"type": "array",
```

Comment:
Unfortunately this is not supported by azure devops. 
However will merge and fix this issue in main branch.